Assets/HoloToolkit/Input/Scripts/InputSources/InputSourceEventArgs.cs

```diff
@@ -1,94 +0,0 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License. See LICENSE in the project root for license information.
-
-using System;
-using UnityEngine;
-
-namespace HoloToolkit.Unity.InputModule
-{
-    /// <summary>
-    /// Basic event args for an input source event.
-    /// </summary>
-    public class InputSourceEventArgs : EventArgs
```

Comment:
Potentially, I'm not an expert on the GC's behavior. We can revisit this is a separate CR,